tests/view/domconverter/dom-to-view.js

```diff
@@ -175,6 +175,15 @@ describe( 'DomConverter', () => {
 			expect( converter.domToView( comment ) ).to.be.null;
 		} );
 
+		it( 'should create namespaced elements', () => {
```

Comment:
AFAICS this test would pass even without this change. We need a test that would fail on master.

Reply:
And the test was in the wrong place. Fixed now and fails without the change :)